src/sagemaker/local/local_session.py

```diff
@@ -98,6 +98,7 @@ def create_training_job(
         )
         training_job = _LocalTrainingJob(container)
         hyperparameters = kwargs["HyperParameters"] if "HyperParameters" in kwargs else {}
+        logger.info("Starting training job")
```

Comment:
another leftover debugging statement?

Reply:
This was more to actually output that things are starting. I don't know what the guidance on logging itself is. Happy to remove it.

Reply:
I think I'm okay with this one then. iirc training on SageMaker prints a similar message